tests/tests/test_clickhouse.py

```diff
@@ -51,71 +51,71 @@ def test_ch_001(self):
     create_mv3 = "create materialized view t_mv3 on cluster default to t3 as select a from t1"
 
     with Given("Tables t1, t2, t3 and MVs t1->t2, t1-t3 are created"):
-        clickhouse.query(chi, create_table)
-        clickhouse.query(chi, create_mv_table2)
-        clickhouse.query(chi, create_mv_table3)
+        clickhouse.query(self.context.runner, chi, create_table)
```

Comment:
why we need `self.context.runner`
instead of just redefine kubectl_cmd in `settings.py`?